frontend/js/pages/CardPokemon.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
```

Comment:
What do you think of putting this in the components folder?

Reply:
great!